gem2rpm/foreman_plugin.spec.erb

```diff
@@ -48,7 +48,7 @@ Summary: <%= spec.summary.gsub(/\.$/, "") %>
 Group: Applications/Systems
 License: <%= spec.licenses.empty? ? 'FIXME' : spec.licenses.map { |l| LICENSES.fetch(l, l) }.join(" and ") %>
 <% if spec.homepage -%>
-URL: <%= spec.homepage %>
+URL: <%= spec.homepage.include?('http://github.com') ? spec.homepage.gsub('http://', 'https://') : spec.homepage %>
```

Comment:
I wonder if `spec.homepage` is ever `nil`. Other than that I still like fixing it in upstream gemspecs to improve the ecosystem a bit.
```suggestion
URL: <%= spec.homepage.start_with?('http://github.com') ? spec.homepage.gsub('http://', 'https://') : spec.homepage %>
```